telegram/_bot.py

```diff
@@ -3412,7 +3412,7 @@ async def edit_message_reply_markup(
         chat_id: Optional[Union[str, int]] = None,
         message_id: Optional[int] = None,
         inline_message_id: Optional[str] = None,
-        reply_markup: Optional["InlineKeyboardMarkup"] = None,
+        reply_markup: Optional[InlineKeyboardMarkup] = None,
```

Comment:
It's interesting that in some places you are converting normal references into ForwardRefs, and here the direction is opposite.

Reply:
was a consequence of an old approach, now they can just continue to be ForwardRefs